### DISCUSSION TITLE
Support for NativeAOT for newer extensions in Semantic Kernel

### DISCUSSION THREAD

**Discussion by tekstop** [microsoft/semantic-kernel#9937] [Q&A] [answered]
I noticed that semantic-kernel recently added support for NativeAOT. Would this support be maintained for the newer features released like the Process Framework and Agents?
We are considering using NativeAOT as a possible choice for deployment and wanted to make sure that this would be continued to be supported.

**Comment by moonbox3** [comment]
Tagging @markwallace-microsoft for visibility.

**Comment by markwallace-microsoft** [comment, marked as answer]
@tekstop We have enabled Native AOT for a subset of SK packages so far but plan to extend this next year and we will maintain this support going forward. It would be great if you could join our community office hours to discuss your requirements, see https://github.com/microsoft/semantic-kernel/blob/main/COMMUNITY.md
